### DISCUSSION TITLE
How to configure the Windows package to use a permanent Session

### DISCUSSION THREAD

**Discussion by DavidTBrookes** [FlareSolverr/FlareSolverr#893] [Q&A]
How do you configure the Window package to use a permanent session and then how to close it when finished with it?

**Comment by Elzeus19** [comment]
i kinda want to know too. I use the source code for windows

**Reply by TapeMeOh** [reply]
Did you ever get an answer to this?

**Comment by ilike2burnthing** [comment]
https://github.com/FlareSolverr/FlareSolverr#commands
